cmd/build/command.go

```diff
@@ -215,31 +214,6 @@ func getOktetoContext(ctx context.Context, options *types.BuildOptions) (*okteto
 		Show:      true,
 	}
 
-	// before calling the context command, there is need to retrieve the context and
-	// namespace through the given manifest. If the manifest is a Dockerfile, this
-	// information cannot be extracted so call to GetContextResource is skipped.
-	if err := validateDockerfile(options.File); err != nil {
```

Comment:
we don't need to detect dockerfile anymore because there is no need to read `context` and `namespace` from the manifest anymore